package.json

```diff
@@ -13,6 +13,7 @@
     "./use-at-your-own-risk": "./lib/unsupported-api.js"
   },
   "scripts": {
+    "build:docs:check-rule-examples": "node Makefile.js checkRuleExamples",
```

Comment:
Per https://eslint.org/docs/latest/contribute/package-json-conventions, `build` scripts generate files, while this only checks existing files. Maybe `lint:`?

Reply:
Thanks for the link. I've renamed the script to `lint:docs:rule-examples` because `lint:docs:check-rule-examples` sounds pleonastic: https://github.com/eslint/eslint/pull/17718/commits/59335823f195c2870dbbcf850774540f8c993a9f.